source/Ship.cpp

```diff
@@ -3533,37 +3533,65 @@ double Ship::TurnRate() const
 
 
 
-double Ship::Acceleration() const
+// Get maximum forward thrust. Use regular thruster and afterburner if allThrust is set.
+// Otherwise use regular thrusters if they are available and afterburner only if not.
+double Ship::ForwardThrust(bool allThrust) const
 {
 	double thrust = attributes.Get("thrust");
-	return (thrust ? thrust : attributes.Get("afterburner thrust")) / InertialMass();
+	if (allThrust)
```

Comment:
```suggestion
	if(allThrust)
```

Reply:
Thanks @warp-core! Turned out, that my browser filtered out the necessary JavaScript which included the information.